lib/OnboardingMessageProvider.jsm

```diff
@@ -25,8 +25,12 @@ async function getAddonName() {
         break;
       }
     }
-    const addons = await AddonRepository.getAddonsByIDs([content]);
-    return addons[0].name;
+    const [addon] = await AddonRepository.getAddonsByIDs([content]);
+    return {
+      name: addon.name,
+      url: addon.sourceURI.spec,
```

Comment:
I realize this is code moved from ASRouter.jsm, but could we check that sourceURI.scheme === "https"